xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -22,9 +22,15 @@
 
 namespace
 {
+
+// Directory where user defined fonts are located (and where mkv fonts are extracted to)
+constexpr const char* userFontPath = "special://home/media/Fonts/";
+// Directory where Kodi bundled fonts (default ones like Arial or Teletext) are located
+constexpr const char* systemFontPath = "special://xbmc/media/Fonts/";
+
 std::string GetDefaultFontPath(std::string& font)
 {
-  std::string fontSources[]{"special://home/media/Fonts/", "special://xbmc/media/Fonts/"};
+  const std::array<const char*, 2> fontSources{{userFontPath, systemFontPath}};
```

Comment:
```suggestion
  constexpr std::array<const char*, 2> fontSources{userFontPath, systemFontPath};
```

constexpr and I don't think you need the double braces

Reply:
done, thanks.
I'll merge once jenkins passes